docs/dev/core/internal/implementations.md

```diff
@@ -0,0 +1,17 @@
+# Implementations
+
+| Project Name      | Language Bindings   | Operating System  | App lifecycle type  | Environment Data source |
+| ----------------- | ------------------- | ----------------  | ------------------- | ----------------------- |
+| glean-core        | Rust                | all               | all                 | none                    |
+| glean-ffi         | C                   | all               | all                 | none                    |
+| glean-preview[^1] | Rust                | Windows/Mac/Linux | Desktop application | OS info build-time autodetected, app info passed in |
+| Glean Android     | Kotlin, Java        | Android           | Mobile app          | Autodetected from the Android environment |
+| Glean iOS         | Swift               | iOS               | Mobile app          | Autodetected from the iOS environment
+| Glean.py          | Python              | Windows/Mac/Linux | all                 | Autodetected at runtime |
+| FOG[^2]           | Rust/C++/JavaScript | as Firefox supports | Desktop application | OS info build-time autodetected, app info passed in |
```

Comment:
I'm torn about having this row here, but the note makes me a bit less opinionated.

If others agree about keeping this in, specifically @mdboom , then I'm fine.

Reply:
you mean listing FOG?
I was too, but at the moment this would be the best place to list it to keep it all together (plus right now everything in that list is maintained but us)